pkg/splunk/enterprise/licensemaster.go

```diff
@@ -48,6 +48,17 @@ func ApplyLicenseMaster(client splcommon.ControllerClient, cr *enterprisev1.Lice
 		client.Status().Update(context.TODO(), cr)
 	}()
 
+	// create or update general config resources
+	_, err = ApplySplunkConfig(client, cr, cr.Spec.CommonSplunkSpec, SplunkLicenseMaster)
```

Comment:
Why do we now reconcile before the "CheckForDeletion" check? Wouldn't it mean we are reconciling a deleted CR?

Reply:
To update MC configMap in the deletion case we need to call ApplyMonitoringConsole() with "cr.ObjectMeta.DeletionTimestamp != nil" , after the deployment is updated will just delete the CR as usual and return